theano/gof/graph.py

```diff
@@ -822,7 +822,7 @@ def clone(i, o, copy_inputs=True):
     return [equiv[input] for input in i], [equiv[output] for output in o]
 
 
-def clone_get_equiv(inputs, outputs, copy_inputs_and_orphans=True, memo=None):
+def clone_get_equiv(inputs, outputs, copy_inputs=True, copy_orphans=True, memo=None):
```

Comment:
Haha, but changing the behaviour of `clone` was exactly what I wanted to do! It's behaviour was very weird in a sense that `copy_inputs=False` also prevented constants from being copied. Are you sure that you want to keep this behaviour? The tests pass even with this change.